src/ui/components/DayBox.tsx

```diff
@@ -0,0 +1,56 @@
+import styleTokenCss from '@ui/styles/styleToken.css';
+import styled from '@emotion/styled';
+
+const calendarImageTypeSrc = {
+  today: 'images/icon/calendar/today.svg',
+  available: 'images/icon/calendar/available.svg',
+  disabled: 'images/icon/calendar/disabled.svg',
+  great: 'images/icon/calendar/feeling-cat-angry.svg',
+  good: 'images/icon/calendar/feeling-cat-good.svg',
+  normal: 'images/icon/calendar/feeling-cat-normal.svg',
+  bad: 'images/icon/calendar/feeling-cat-bad.svg',
+  angry: 'images/icon/calendar/feeling-cat-angry.svg',
+};
+
+type DayBoxProps = {
+  date: number;
+  calendarType: 'today' | 'available' | 'disabled' | 'great' | 'good' | 'normal' | 'bad' | 'angry';
```

Comment:
```suggestion
  type: 'today' | 'available' | 'disabled' | 'great' | 'good' | 'normal' | 'bad' | 'angry';
```
P3:
날짜를 입력하는 컴포넌트라는 네이밍으로 구성되어 있어서
props를 type으로만 선언해주어도 다른 별도의 타입이 있는것이 아니기 때문에 괜찮을것 같아요.